src/SprykerEco/Yves/Heidelpay/Form/CreditCardSecureSubForm.php

```diff
@@ -38,8 +66,7 @@ public function configureOptions(OptionsResolver $resolver): void
     {
         $resolver->setDefaults([
             'data_class' => HeidelpayCreditCardPaymentTransfer::class,
-            SubFormInterface::OPTIONS_FIELD_NAME => [],
-        ]);
+        ])->setRequired(static::OPTIONS_FIELD_NAME);
```

Comment:
Do we really need them as required field everywhere?

Reply:
Yes we do.